netbox_onboarding/templates/netbox_onboarding/onboardingtask.html

```diff
@@ -0,0 +1,91 @@
+{% extends 'base.html' %}
+{% load buttons %}
+{% load custom_links %}
+{% load helpers %}
+{% load static %}
+{% load tz %}
```

Comment:
That's a lot of `load`s here - are they all actually needed for such a simple page?

Reply:
Fixed